### PR TITLE
Change `Flexible`'s default `FlexFit`

### LINKED ISSUE #5169: Rename Flexible to Expand and Fit
We've received feedback from new users that our Flex implementation is kinda-but-not-really the Flexbox they know and love from the web. This can cause a bit of friction as they learn how our Flex is and is not Flexbox.

We can consider renaming Flex to help avoid users drawing parallels and miss-set expectations. One idea was the rename to Linear.

Presumably this would be a breaking change and if we do this, we can email our users, etc.

### PR DESCRIPTION
**Pull request by abarth**
Previously, `Flexible` defaulted to `FlexFit.tight`, which forced the child to
expand to fill the available space. Now, `Flexible` defaults to
`FlexFit.loose`, which does not force the child to expand to fill the available
space.

If you want the child to expand to fill the available space, consider using
`Expanded` instead.

Fixes #5169

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie @HansMuller

File: packages/flutter/lib/src/widgets/basic.dart [hunk 1]
```diff
@@ -2229,17 +2229,27 @@ class Column extends Flex {
 
 /// A widget that controls how a child of a [Row], [Column], or [Flex] flexes.
 ///
+/// Using a [Flexible] widget to gives a child of a [Row], [Column], or [Flex]
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/basic.dart]: s/to//

File: packages/flutter/lib/src/widgets/basic.dart [hunk 2]
```diff
@@ -2298,9 +2308,13 @@ class Flexible extends ParentDataWidget<Flex> {
 /// the available space is divided amoung them according to the [flex] factor.
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/basic.dart]: s/Using/Use/ or s/to make/makes/.

**Comment by Hixie** [comment]
I'm a little concerned that flipping the default broke no tests, but, LGTM!

**Comment by abarth** [comment]
> I'm a little concerned that flipping the default broke no tests, but, LGTM!

I did a previous CL where I migrated everyone to `Expanded`.

**Comment by Hixie** [comment]
Sure but that means there was no test verifying that not giving any arguments to Flexible meant it was treated like Expanded. Or vice versa now. :-)

**Comment by abarth** [comment]
There were previously, but I guess I removed them when I migrated them to `Expanded`.  It's fair that I should add one back now given that I'm changing behavior in this patch.

**Pull request merged**